### ISSUE TITLE
debian extension install fails

### ISSUE THREAD

**Issue by lucapaganotti** [kkrisz1/staruml-reverse-db#45]
Trying to install the staruml-reverse-db extension on my debian box fails, giving me an "Unknown internal error". Am I missing something?

StarUML V3.1.0
debian stretch 9.9

java -version
openjdk version "1.8.0_212"
OpenJDK Runtime Environment (build 1.8.0_212-8u212-b01-1~deb9u1-b01)
OpenJDK 64-Bit Server VM (build 25.212-b01, mixed mode)

javac -version
javac 1.8.0_212

Hope this helps.

**Comment by kkrisz1** [comment]
Could you paste the error logs?
https://docs.staruml.io/developing-extensions/getting-started#run-and-debug

**Comment by lucapaganotti** [comment]
Hi, thanks for your prompt reply, I was in a meeting, now back to my
desktop. Not sure I have done the right thing. In the console of debug
devTools I get this (see image attached).

----------------------------------------------------------------
-- Dott. Ing. Luca Paganotti
-- Via dei Giardini 9
-- 21035 Cunardo (VA)
-- 393 1346898
----------------------------------------------------------------
-- softech s.r.l. email:
-- luca.paganotti@softechweb.it
-- luca.paganotti@gmail.com
-- https://github.com/lucapaganotti
-- sourceforge email:
-- lucapaganotti@users.sourceforge.net
-- skype name: luca.paganotti
[image: http://it.linkedin.com/in/lucapaganotti]
<http://it.linkedin.com/in/lucapaganotti>
-- Mastodon: l <https://www.facebook.com/luca.paganotti.66>
ucapaganotti@fosstodon.org
-- ---------------------------------------------------------------
-- Mistakes are portals of discovery - JAAJ
--- --------------------------------------------------------------

**Comment by kkrisz1** [comment]
cannot find the image.
you can copy and paste the content of the debug console

**Comment by lucapaganotti** [comment]
Oh, sorry, I replied directly from my email client and attached the
screenshot, maybe this the reason, I'll try to put it directly on github
... sorry again

----------------------------------------------------------------
-- Dott. Ing. Luca Paganotti
-- Via dei Giardini 9
-- 21035 Cunardo (VA)
-- 393 1346898
----------------------------------------------------------------
-- softech s.r.l. email:
-- luca.paganotti@softechweb.it
-- luca.paganotti@gmail.com
-- https://github.com/lucapaganotti
-- sourceforge email:
-- lucapaganotti@users.sourceforge.net
-- skype name: luca.paganotti
[image: http://it.linkedin.com/in/lucapaganotti]
<http://it.linkedin.com/in/lucapaganotti>
-- Mastodon: l <https://www.facebook.com/luca.paganotti.66>
ucapaganotti@fosstodon.org
-- ---------------------------------------------------------------
-- Mistakes are portals of discovery - JAAJ
--- --------------------------------------------------------------

**Comment by lucapaganotti** [comment]
Here is the image you requested.
![Schermata da 2019-05-21 12-06-37](https://user-images.githubusercontent.com/6130461/58713421-d57ace00-83c2-11e9-9603-351b1200c7fe.png)
Hope this helps.

**Comment by kkrisz1** [comment]
Have not seen this error before

**Comment by kkrisz1** [comment]
could you try to install the extension manually?

[Install Extension](https://docs.staruml.io/user-guide/managing-extensions#install-extension)
![image](https://user-images.githubusercontent.com/19638316/59267916-b8a88b00-8c4b-11e9-82fc-6b27691e384c.png)

**Comment by lucapaganotti** [comment]
Hi, got the zip file cloning this repository. Unpacked it, after closing staruml in the ~/.config/StarUML/extensions/user folder. Run again staruml, check plugin status and I get this. See attachment.

![staruml-reverse-db-install-fails](https://user-images.githubusercontent.com/6130461/59270336-839f3700-8c51-11e9-982d-d03a60b5e7f7.png)

**Comment by kkrisz1** [comment]
Please, download and install the [staruml-reverse-db-1.0.11.zip](https://github.com/kkrisz1/staruml-reverse-db/releases/download/v1.0.11/kkrisz1.staruml-reverse-db-1.0.11.zip)

**Comment by kkrisz1** [comment]
close the issue, because there has been no answer for 1 week

**Comment by lucapaganotti** [comment]
I'm very sorry, but I can answer only today. I've installed you extension many times, without success. Each time I unzipped the extension zip file in a folder called staruml-reverse-db under the proper path in .config then checked the presence of main.js file. The last time I leaved unchanged the extension folder name which became, after unzipping, kkrisz1.staruml-reverse-db-1.0.11. It seems, at least on my box, that the folder name matters. Now the extension is installed, even if marked with issues. Thank you for your time, I hope this can help.
